File: clanguages/Maximum.c
#include <stdio.h>


int main(void) {
  
  int Counter = 0, Size;
  printf("enter a number:");
  scanf("%d", &Size);
  int Numbers[Size];
  printf("To find the greatest number amoung %d numbers, enter the first number:", Size);
  scanf("%d", &Numbers[Counter]);
  for(Counter = 1; Counter < Size; Counter++) {
    printf("Enter next number:");
    scanf("%d", &Numbers[Counter]);
  }
  int Max = Numbers[0];
  for(Counter = 1; Counter < Size; Counter++) {
    if(Max < Numbers[Counter]) {
      Max = Numbers[Counter];
    }
  }
  printf("The greatest of %d numbers is %d.",Size, Max);
  return 0;
}

File: pyhton/Maximum.py
ReadList = []
ReadList.append(int(input("To find the greatest number amoung ten numbers, enter the first number:")))
for Counter in range(1, 10):
  ReadList.append(int(input("Enter next number: ")))
Max = ReadList[0]
for Counter in range(1, 10):
  if Max < ReadList[Counter]:
    Max = ReadList[Counter]
print("The greatest of ten numbers is {}.".format(Max))
  